ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,37 @@
-<!-- Guidelines for quest suggestions: https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete -->
+<!--
+If you are not suggesting a new quest, ignore this issue template. Otherwise we kindly ask you to fill it out.
 
+Guidelines for quest suggestions: https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete
+-->
+
+### General
+Affected tag(s) to be modified: [?????](https://wiki.openstreetmap.org/wiki/Tag:?????)
+Question asked: **Is there an example tag on item XY?**
+
+<!-- If the answer is not obvious, please append a short explanation to each item (or, if it is longer, below), explaining the reason. -->
+### Checklist
 Checklist for quest suggestions (see guidelines):
 - [ ] 🚧 To be added tag is established and has a useful purpose
-- [ ] 🤔 Any answer the user can give must have an equivalent tagging (No false-positives) 
+- [ ] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
 - [ ] 🐿️ Easily answerable by everyone from the outside
 - [ ] 💤 Not an overwhelming percentage of elements have the same answer (No spam)
 - [ ] 🕓 Applies to a reasonable number of elements (Worth the effort)
+
+<!--
+Please try to satisfy all conditions. If one cannot be satisfied that may mean your quest will not be included in StreetComplete.
+If you are not sure about how one condition applies to your suggestion or you have (very) strong reasons that a condition may not needed to be satisfied in your case, just note that down. Other people may come up with some ideas.
+-->
+
+### Ideas for implementation
+<!-- If you do not have any, just skip this part. ;)  -->
+
+<!-- If you have any idea for how elements should be selected, add it here. Possibly include an overpass-api query if you are sure what you do. -->
+**Element selection:** 
+
+<!-- If you have any idea whether metadata per country is needed, add it here. -->
+**Metadata needed:** 
+
+<!-- If you have any idea on how the GUI (the from appearing when the quest is asked) should look like. (i.e. whether a simple yes/no answer is suitable or what else you imagine) You can use any way to propose a UI. Use simple markdown tables or complex image mockups. That's your decicion.
```

Comment:
The url https://help.github.com/articles/organizing-information-with-tables/ should be near the word "Markdown Table"